lib/Runtime/Library/JavascriptBigInt.h

```diff
@@ -4,26 +4,26 @@
 //-------------------------------------------------------------------------------------------------------
 #pragma once
 
+
 namespace Js
 {
     class JavascriptBigInt sealed : public RecyclableObject
     {
-
     private:
-        Field(uint32*) m_digits;         // digits
-        Field(uint32) m_length;          // length
+        Field(digit_t*) m_digits;         // digits
+        Field(digit_t) m_length;          // length
+        Field(digit_t) m_maxLength;          // max length without resize
         Field(bool) m_isNegative;
 
         // TODO: BigInt should be arbitrary-precision, need to resize space and add tagged BigInt
-        static const uint32 MaxStringLength = 100; // Max String length
-        static const uint32 MaxDigitLength = 20;  // Max Digit length 
-        Field(uint32) m_reservedDigitsInit[MaxDigitLength]; // pre-defined space to store array
+        static const digit_t InitDigitLength = 20;  // Max Digit length 
```

Comment:
This is rather a lot of space to allocate by default. BigInt is pretty new and not a whole lot of code uses it yet, but I would guess that the majority of BigInts fit within 2 digits.

Reply:
let it be 2 then 👍